plugin/storage/cassandra/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM cassandra:3.11
-
-COPY schema/* /cassandra-schema/
-
-ENV CQLSH_HOST=cassandra
-ENTRYPOINT ["/cassandra-schema/docker.sh"]
+ENV DATACENTER=dc1 KEYSPACE=jaeger_v1_dc1 MODE=prod
+COPY schema/create.sh /
+COPY schema/v001.cql.tmpl /
+COPY schema/entrypoint-wrapper.sh /
+ENTRYPOINT ["/entrypoint-wrapper.sh"]
+CMD ["cassandra", "-f"]
```

Comment:
The reason I did this was to avoid extra complexity Cassandra image. I might be able to do this in an entirely separate image (cassandra-schema), so I'll revisit that idea.